### PR TITLE
8305084: Remove the removal warnings for finalize() from test/hotspot/jtreg/serviceability/dcmd/gc/FinalizerInfoTest.java and RunFinalizationTest.java

### PR DESCRIPTION
**Pull request by afshin-zafari**
The `removal` warnings are suppressed out.
Test:
`FinalizerInfoTest` and `RunFinalizationTest` are executed locally.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8305084](https://bugs.openjdk.org/browse/JDK-8305084): Remove the removal warnings for finalize() from test/hotspot/jtreg/serviceability/dcmd/gc/FinalizerInfoTest.java and RunFinalizationTest.java

### Reviewers
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**)
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13423/head:pull/13423` \
`$ git checkout pull/13423`

Update a local copy of the PR: \
`$ git checkout pull/13423` \
`$ git pull https://git.openjdk.org/jdk.git pull/13423/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13423`

View PR using the GUI difftool: \
`$ git pr show -t 13423`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13423.diff">https://git.openjdk.org/jdk/pull/13423.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13423#issuecomment-1503073448)

### CODE REVIEW

File: test/hotspot/jtreg/serviceability/dcmd/gc/FinalizationRunner.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```
Comment (dholmes-ora) [inline test/hotspot/jtreg/serviceability/dcmd/gc/FinalizationRunner.java:2]: Copyright change is wrong - should be "2015, 2023,"

**Review by dholmes-ora — CHANGES_REQUESTED** [review]
Suppression of warning is fine but the bug synopsis (and so PR title) needs to be updated to reflect what is actually happening here. And the bug title doesn't really make sense anyway - why is the test location given twice ??

Thanks

**Comment by afshin-zafari** [comment]
> Suppression of warning is fine but the bug synopsis (and so PR title) needs to be updated to reflect what is actually happening here. And the bug title doesn't really make sense anyway - why is the test location given twice ??

Bug and PR titles are changed.

**Review by dholmes-ora — APPROVED** [review]
Looks good. Thanks.

**Comment by afshin-zafari** [comment]
Thank you David and Chris for your reviews.
/integrate

**Pull request closed without merging**